perun-core/src/main/java/cz/metacentrum/perun/core/impl/ResourcesManagerImpl.java

```diff
@@ -1216,6 +1217,30 @@ public void removeBan(PerunSession sess, int memberId, int resourceId) throws Ba
 		}
 	}
 
+	@Override
+	public GroupResourceStatus getGroupResourceStatus(PerunSession sess, Group group, Resource resource) throws GroupNotDefinedOnResourceException {
+		try {
+			String status = jdbc.queryForObject("select status from groups_resources where group_id=? and resource_id=?",
+				String.class, group.getId(), resource.getId());
+			return GroupResourceStatus.valueOf(status);
+		} catch (EmptyResultDataAccessException ex) {
+			throw new GroupNotDefinedOnResourceException("Group " + group.getId() + " is not defined on resource " + resource.getId());
+		} catch (RuntimeException e) {
+			throw new InternalErrorException(e);
+		}
+	}
+
+	@Override
+	public void setGroupResourceStatus(PerunSession sess, Group group, Resource resource, GroupResourceStatus status) {
+		try {
+			jdbc.update("update groups_resources set status=?::group_resource_status, modified_by=?, modified_by_uid=?, " +
+					" modified_at=" + Compatibility.getSysdate() + " where group_id=? and resource_id=?", status.toString(),
```

Comment:
Since we no longer support multiple DB types, you can simplify this:
```suggestion
					" modified_at=statement_timestamp() where group_id=? and resource_id=?", status.toString(),
```

Reply:
Replaced.